core/src/main/java/org/testcontainers/containers/wait/internal/InternalCommandPortListeningCheck.java

```diff
@@ -29,7 +29,7 @@ public Boolean call() {
         for (int internalPort : internalPorts) {
             command += " && ";
             command += " (";
-            command += format("cat /proc/net/tcp{,6} | awk '{print $2}' | grep -i :%x", internalPort);
+            command += format("cat /proc/net/tcp* | awk '{print $2}' | grep -i :%x", internalPort);
```

Comment:
could you please add a test for this change?

Reply:
@bsideup I would love to, and I tried to think up a way to add a test (preferably ones that fails before this change) test, but honestly, it's very hard to come up with one. 🙂 One of the problems is that e.g. most (all?) Alpine images (like `nginx:1.17-alpine`) have a working `busybox` which handles the `nc` invocation => it's really hard to verify that the "old" version fails (and that the "new" version works) on these images.

It's similar with the Debian-based ones, but differently: here the `nc` invocation fails but the latter part (`/bin/bash -c '</dev/tcp/localhost/80'`) catches it...

To make this more clear, this is what the command line outputs with that container with our existing (non-dash-compatible) expansion. The `localhost (127.0.0.1:80) open` line comes from the `nc` invocation => the check correctly detects the port correctly as being listening to.

```shell
/ # true &&  (cat /proc/net/tcp{,6} | awk '{print $2}' | grep -i :0050 || nc -vz -w 1 localhost 80 || /bin/bash -c '</dev/tcp/localhost/80')
cat: can't open '/proc/net/tcp{,6}': No such file or directory
localhost (127.0.0.1:80) open
```

With `nginx:1.9.4` (which seems to be Debian-based), it looks like this:

```shell
# true &&  (cat /proc/net/tcp{,6} | awk '{print $2}' | grep -i :0050 || nc -vz -w 1 localhost 80 || /bin/bash -c '</dev/tcp/localhost/80')
cat: /proc/net/tcp{,6}: No such file or directory
/bin/sh: 8: nc: not found
# echo $?
0
```

In this case, the `/bin/bash -c '</dev/tcp/localhost/80'` detects the service listening to the port.

If I get really nasty and do `dpkg --purge --force-all bash` in the container, I can make the test fail completely (with the Debian-based `nginx:1.9.4` image), but clearly, there must be a more elegant and reasonable way. :slightly_smiling_face: Do you have any ideas?

Reply:
@bsideup Ping, please let me know if you have any ideas. Would love to add a test for this if we can just do it with reasonable effort.

Reply:
I think this is reasonable. It's not ideal, but the change is obvious and we can see that it's not causing unexpected regressions in existing tests.

Reply:
:+1: